src/screens/LinkContextScreen/elements/Header.js

```diff
@@ -226,7 +226,7 @@ const styles = StyleSheet.create({
     marginLeft: -4
   },
   wrapperText: {
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the wrapperText component has been changed from `COLORS.white` to `COLORS.almostBlack`. As with previous comments, ensure this change aligns with your overall design system and does not compromise the readability of any text within this component.
```diff
- backgroundColor: COLORS.white
+ backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->